runner/Paperbag/Window.cpp

```diff
@@ -6,9 +6,6 @@ using namespace paperbag;
 Window::Window(Settings s)
     : settings(std::move(s))
 {
-    if constexpr (!opengl_loader_gl3w)
```

Comment:
you should probably remove this constexpr variable now :)

Reply:
It has been removed

Reply:
using it yah, but it's still declared